README.md

```diff
@@ -100,14 +100,14 @@ If the PR was closed without comments it's likely due to QA failure. If the Gith
 
 __Q: the compiler/code analyser/tool is wrong, what now?__
```

Comment:
Could you change the initial letter to capital, like you did elsewhere, too?

Reply:
Unfortunately I have deleted the fork so I'm not sure how I'm supposed to edit the file now. Is there any way to do so?

Reply:
Okay, I'll have to remake then, how do I link the new changes to this pull request, do I have to make a new pull request as well?

Reply:
In general, we prefer that changes are incorporated into new pull requests, so this would the way, yes :)

Reply:
Great, I'll close this pull request and make a new one with the appropriate changes.